uxarray/__init__.py

```diff
@@ -1,2 +1,17 @@
 from .grid import *
 from .helpers import *
+
+
+def __version__():
+    # Attempt to import the needed modules
+    try:
+        from importlib.metadata import version as version
+    except ImportError:
+        from importlib_metadata import version as version
+
+    try:
+        __version = version("uxarray")
+        print(__version)
+    except AttributeError:
```

Comment:
This should be a `NameError`, since if our library isn't installed, we would have never declared the `version` method above. 

Attempting to execute `version("uxarray")` would then raise a `NameError`

Reply:
I think we can actually generalize this to just `except Exception`, as we don't really care what specifically goes wrong, just that something did.

Reply:
Good point, thanks for all the feedback on everything, I really appreciate it!